File: SmallPrograms/fibonacci/fibonacci.c
#include <stdio.h>
struct llv
{
	long long a;
	long long b;
};
struct llv fib(int n)
{
	struct llv ret = {1, 1};
	if(n==2||n==1)
		return ret;
	ret = fib(n-1);
	ret.b += ret.a;
	ret.a = ret.b - ret.a;
	return ret;
}
int main()
{
	for(int i=1;i<=90;i++)
		printf("%d\t%lld\n", i, fib(i).b);
}

File: SmallPrograms/fibonacci/fibonacci.py
def fib(n):
	if n==2 or n==1:
		return (1, 1)
	else:
		a, b = fib(n-1)
		return (b, a+b)

for i in range(1, 91):
	a, b = fib(i)
	print(i, b)